### PR TITLE
Konstantinos / [FE Deriv.com squad] [EU] Review and update the design of Dtrader page

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]
LGTM

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             )}
             <StyledFlexGridContainer content_width="38.4rem" gap="1rem" grid="3" justify="center">
                 {exclude.toLowerCase() !== 'dtrader' && <TraderCard />}
-                <ROW>{exclude.toLowerCase() !== 'dbot' && <BotCard />}</ROW>
+                {exclude.toLowerCase() !== 'dbot' && <BotCard />}
                 {exclude.toLowerCase() !== 'dmt5' && <DMT5Card is_ppc_redirect={is_ppc_redirect} />}
```
Comment (prince-deriv) [inline src/components/custom/other-platforms.js:235]: Not part of your change but can we move `exclude.toLowerCase()` to variable instead?

File: src/pages/dtrader/_dtrader-tabs.tsx [hunk 2]
```diff
@@ -278,7 +278,10 @@ class DtraderTabs extends React.Component {
                             </>
                         </ROW>
                         <EU>
-                            <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
+                            <>
+                                <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
+                                <source src="/Dtrader_GIF_EU.webm" type="video/webm" />
+                            </>
                         </EU>
```
Comment (prince-deriv) [inline src/pages/dtrader/_dtrader-tabs.tsx:285]: Can this work without the react fragment `<>`?

```
<EU>
  <source src="/Dtrader_GIF_EU.mp4" type="video/mp4" />
  <source src="/Dtrader_GIF_EU.webm" type="video/webm" />
</EU>
```

Comment (konstantinosG-deriv) [reply src/pages/dtrader/_dtrader-tabs.tsx:285]: If I remove the fragment I get this error: This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor>', but multiple children were provided.

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (prince-deriv) [reply src/components/custom/other-platforms.js:235]: make it `const` instead also it is for readability and ease of development, later you will only need to update it in one place.

File: src/pages/dtrader/_dtrader-tabs.tsx (hunk 2 above)
Comment (prince-deriv) [reply src/pages/dtrader/_dtrader-tabs.tsx:285]: Ok keep it like that then

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/components/custom/other-platforms.js:235]: Ok

File: src/components/custom/other-platforms.js [hunk 3]
```diff
@@ -233,7 +233,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             <StyledFlexGridContainer content_width="38.4rem" gap="1rem" grid="3" justify="center">
                 {}
                 {excludetoLowerCase !== 'dtrader' && <TraderCard />}
-                {excludetoLowerCase !== 'dbot' && <BotCard />}
+                <ROW>{excludetoLowerCase !== 'dbot' && <BotCard />}</ROW>
```
Comment (prince-deriv) [inline src/components/custom/other-platforms.js:236]: can we `use is_row` instead? 

```suggestion
                {excludetoLowerCase !== 'dbot' && is_row && <BotCard />}
```

Comment (konstantinosG-deriv) [reply src/components/custom/other-platforms.js:236]: Ok

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**